oar/core/advisory_mgr.py

```diff
@@ -397,3 +413,46 @@ def are_all_push_jobs_completed(self):
                 completed = False
 
         return completed
+
+    def is_doc_approved(self):
+        """
+        Check if doc is approved for a advisory
+        Returns:
+        bool: True if doc for a advisory is approved, otherwise False
+        """
+        appr_status = self.get_erratum_data()["doc_complete"]
+        if appr_status == 1:
+            return True
+        return False
+    
+    def is_prodsec_approved_or_requested(self):
```

Comment:
suggest to split `approved` and `requested` to different funcs

Reply:
could you check again, thanks